packages/workbox-webpack-plugin/src/lib/get-webpack-manifest-entries.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
+
+const entry = (revision, filePath, publicPath) => ({
+  revision,
+  url: url.resolve(publicPath, filePath),
+});
+
+/**
+ * Filter chunks that have one of the provided names (from config.chunks)
+ * @param {Array<Object>} chunks Webpack chunks
+ * @param {Array<string>} include Chunk names to include
+ * @param {Array<string>} exclude Chunk names to exclude
+ * @return {Array<Object>} Filtered array of chunks
```

Comment:
Once we have some docs together (for a more fleshed out version), it would be great for them to capture a few different examples of using chunks, include and exclude.

Reply:
+1